src/locking/LockingHelper.ts

```diff
@@ -24,9 +16,9 @@ export default class LockingHelper {
     return lock;
   }
 
-  public static async createSiteAreaSmartChargingLock(tenantID: string, siteArea: SiteArea): Promise<Lock | null> {
-    const lock = LockingManager.createExclusiveLock(tenantID, LockEntity.SITE_AREA, `${siteArea.id}-smart-charging`);
-    if (!(await LockingManager.acquire(lock))) {
+  public static async createSiteAreaSmartChargingLock(tenantID: string, siteArea: SiteArea, timeout: number): Promise<Lock | null> {
```

Comment:
```suggestion
  public static async createSiteAreaSmartChargingLock(tenantID: string, siteArea: SiteArea, timeoutMs: number = 0): Promise<Lock | null> {
```

Reply:
My PR includes this change already and has to be merged if everybody agrees